GetServicePreferences/handler.ts

```diff
@@ -125,6 +146,8 @@ const getUserServicePreferencesOrDefault = (
                 return toDefaultEnabledUserServicePreference(version);
               case ServicesPreferencesModeEnum.MANUAL:
                 return toDefaultDisabledUserServicePreference(version);
+              default:
+                return void 0 as never;
```

Comment:
is this needed? project compiles without the default case because all possible values are already addressed

Reply:
Adding this default case could be beneficial, particularly if enhanced with [Exhaustiveness Checking](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#exhaustiveness-checking)